UT4MasterServer.Web/src/services/http.service.ts

```diff
@@ -9,10 +9,10 @@ export interface HttpRequestOptions<T = unknown> {
 }
 
 export class HttpError {
-  code: number;
+  code: string;
```

Comment:
Shouldn't these always be numbers?

Reply:
Ah, right. The http status code from the fetch api will be a number though, so we should either change the http service to pull numeric error code or change it so that it makes the http error code a string. Otherwise you'll never be able to correctly compare it because typescript will think it has a string when it has a number.

Reply:
I see how you're using it now, I will make some changes to it, don't worry about it

Reply:
We could use `numericErrorCode` instead, but then you would have "magic" numbers on the frontend. I think string representation on an error is better for understanding.
And when I say `errorCode`, I am not referring to the HTTP response code (400 or similar) which is a number.

Reply:
Http status codes are hardly magic numbers, they are well defined, and easily enumerated. And we still have that numeric case, when the error response isn't of type ErrorResponse. I will make it accept either case.